lib/database.rb

```diff
@@ -69,7 +65,7 @@ def self.init
   # 小説格納用のルートディレクトリを取得
   #
   def self.archive_root_path
-    @@archive_root_path
+    @@archive_root_path ||= File.expand_path(File.join(Narou.get_root_dir, ARCHIVE_ROOT_DIR_PATH))
```

Comment:
__[reek]__
* [ClassVariable] declares the class variable @@archive_root_path [:ref](https://github.com/troessner/reek/blob/master/docs/Class-Variable.md)